scripts/gdal/gdalinfo.py

```diff
@@ -0,0 +1,26 @@
+import json
+from typing import Any, Dict
+
+from linz_logger import get_log
+
+from scripts.gdal.gdal_helper import GDALExecutionException, run_gdal
+
+
+def gdal_info(path: str) -> Dict[Any, Any]:
+    gdalinfo_command = ["gdalinfo", "-stats", "-json"]
+
+    try:
+        gdalinfo_process = run_gdal(gdalinfo_command, path)
+        gdalinfo_result = {}
+        try:
```

Comment:
What is the need of having a nested `try/except` here?

Reply:
Uhh, I was trying to replicate the gdalinfo code in non_visual_qa.py
If its unnecessary I can remove it.

Reply:
OK, my bad, I'll refactor that and in `non_visual_qa.py` in another way later. That's ok for now.